dartsim/src/EntityManagement_TEST.cc

```diff
@@ -62,6 +62,12 @@ TEST(EntityManagement_TEST, ConstructEmptyWorld)
   EXPECT_EQ(world, model->GetWorld());
   EXPECT_NE(model, world->ConstructEmptyModel("dummy"));
 
+  auto nestedModel = model->ConstructEmptyModel("empty nested model");
+  ASSERT_NE(nullptr, nestedModel);
+  EXPECT_EQ("empty nested model", nestedModel->GetName());
+  EXPECT_EQ(world, nestedModel->GetWorld());
+  EXPECT_NE(nestedModel, nestedModel->ConstructEmptyModel("dummy"));
+
```

Comment:
Can we add tests for the other methods that were added to `EntityManagementFeatures` (`GetNestedModelCount` and `GetNestedModel` - both the index and name parameter versions)?

Reply:
Added more test expectations in https://github.com/ignitionrobotics/ign-physics/pull/228/commits/212c941319d4a7227ef38ae6bdbf196f2cec576b.